.github/workflows/deploy-cloud-nodes.yml

```diff
@@ -64,7 +64,14 @@ jobs:
             docker pull registry.digitalocean.com/magmo/go-nitro:latest
             docker stop $NODE_NAME || true
             docker rm $NODE_NAME || true
-            docker run --restart=unless-stopped -it -d --name $NODE_NAME -p 3005:3005 -p 4005:4005 -p 80:5005 -e NITRO_CONFIG_PATH=$NITRO_CONFIG_PATH -e SC_PK=$SC_PK -e CHAIN_PK=$CHAIN_PK -v /var/nitro_store:/app/data registry.digitalocean.com/magmo/go-nitro:latest
+            docker run --restart=unless-stopped -it -d --name $NODE_NAME \
```

Comment:
Perhaps to be addressed in another PR, but there is a lot of copy/pasting in this file.

Reply:
Yes I agree. Ideally this would be wrapped in a function within the `.yml` github workflow file. I'm not sure if that's possible. Could also look into copying a `makefile` into each of the droplets and wrapping all these `docker` commands into a single `make` command

Reply:
I propose we make that optimization in a future PR

Reply:
Would composite actions help here? https://docs.github.com/en/actions/creating-actions/creating-a-composite-action

Reply:
I'm not sure that would solve it but perhaps I'm not understanding the full capability of composite actions. The tricky part of our workflow is that we need those docker commands to run within the context of the ssh session inside the Droplet